### ISSUE TITLE
evaluate has low accuracy even mnist.

### ISSUE THREAD

**Issue by wekinu** [jhfjhfj1/autokeras#253]
I tried mnist training in autokeras.
and than compile & fit after reload model. 
The result is about 20 percent identical.
I tried change version 0.2.14 and latest version.
but it's same result .

Can you give me any advise one?

### Setup Details
Include the details about the versions of:
 - OS type and version: Ubuntu 16.04
 - Python: 3.6
 - autokeras: 0.2.18
 - scikit-learn: 0.19.1
 - numpy: 1.14.5
 - keras: 2.2.2
 - scipy: 1.1.0
 - tensorflow: 1.11.0
 - pytorch: 0.4.1

**Comment by jhfjhfj1** [comment]
We don't have this problem.
Can you confirm the version number you use? Which is the latest version?
Thanks.

**Comment by wekinu** [comment]
autokeras==0.2.18

i used 3 ways that
clf.load_searcher().load_best_model().produce_keras_model().save()
and 
export_keras_model() & export_autokeras_model() 


I tried reload model after exported model function
and than
model.compile(optimizer='adam', loss='categorical_crossentropy', metrics=['accuracy'])
model.fit

i think that accuracy is low and no more train.
even mnist data set and custom dataset. it's same situation

**Comment by wekinu** [comment]
this issue solved.
after loaded model, i add it
x = keras_model.output 
x = Activation('softmax', name='activation_add')(x)  
keras_model = Model(keras_model.input, x)    

I refer to another Issue.

but i have another issue.
when training on autokeras with my customed dataset. also another customed dataset with imageclassifier
acc: 96% , val_acc:65%

i think Graphs created with autokeras cause overfitting problems.
What do you think?

**Comment by Npccc** [comment]
I have the same problem.I used the keras model output from autokera to predict the image and found that the accuracy was very low.I added the solution you provided, but still did not solve the problem

**Comment by Npccc** [comment]
I have the same problem.I used the keras model output from autokera to predict the image and found that the accuracy was very low.I added the solution you provided, but still did not solve the problem

**Comment by Npccc** [comment]
I have the same problem.I used the keras model output from autokera to predict the image and found that the accuracy was very low.I added the solution you provided, but still did not solve the problem
